src/assets/icons/nav/feed.svg

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" id="Capa_1" x="0px" y="0px" viewBox="0 0 508 508" style="enable-background:new 0 0 508 508;" xml:space="preserve" width="512px" height="512px">
+<g>
+	<g>
+		<rect x="106.5" y="211.163" width="16.4" height="16.4" fill="#6692a6"/>
+	</g>
+</g>
+<g>
+	<g>
+		<rect x="106.5" y="243.862" width="16.4" height="16.4" fill="#6692a6"/>
+	</g>
+</g>
+<g>
+	<g>
+		<rect x="106.5" y="178.362" width="16.4" height="16.4" fill="#6692a6"/>
+	</g>
+</g>
+<g>
+	<g>
+		<path d="M409.6,138.963v-112.5c0-8.9-4.6-16.8-12.3-21.3c-7.7-4.4-16.9-4.5-24.8,0.1l-202.7,123.9H49.2v32.8H0v114.7h49.2v32.8    h26.1l33.6,161.4c4.3,20.4,22.5,35.3,43.4,35.3c13.4,0,25.9-6,34.4-16.3s11.7-23.9,9-37l-29.9-143.3h4l202.9,123.9l0.2,0.1    c3.8,2.2,8.1,3.3,12.3,3.3c4.2,0,8.4-1.1,12.3-3.3c7.7-4.4,12.3-12.4,12.3-21.3v-112.7c29.9-15.3,49-46.2,49-80.3    S439.7,154.263,409.6,138.963z M49.2,260.263H16.4v-81.9h32.8V260.263z M71.9,293.063h-6.4v-16.4v-114.7v-16.4h98.3v147.5h-1.5    H71.9z M179.6,456.063c1.7,8.3-0.3,16.8-5.7,23.3c-5.3,6.5-13.2,10.3-21.6,10.3c-13.1,0-24.6-9.3-27.3-22.2l-33-158h57    L179.6,456.063z M352.3,401.763l-172-105.1v-154.7l172-105.1V401.763z M393.3,412.163c0,4.3-2.9,6.4-4.1,7.1s-4.4,2.1-8.1,0    l-12.4-7.6v-384.8l12.3-7.5c3.7-2.1,7-0.7,8.2,0c1.2,0.7,4.1,2.8,4.1,7.1V412.163z M409.7,280.463v-122.3    c20.2,13.5,32.8,36.2,32.8,61.2C442.5,244.163,429.9,266.963,409.7,280.463z" fill="#6692a6"/>
+	</g>
+</g>
+<g>
+	<g>
+		<rect x="475.2" y="211.163" width="32.8" height="16.4" fill="#6692a6"/>
+	</g>
+</g>
+<g>
+	<g>
+		<rect x="446.163" y="113.137" transform="matrix(0.7071 -0.7071 0.7071 0.7071 49.6833 362.6205)" width="32.8" height="16.4" fill="#6692a6"/>
+	</g>
+</g>
+<g>
+	<g>
+		<rect x="454.36" y="300.941" transform="matrix(0.707 -0.7072 0.7072 0.707 -88.8992 420.1034)" width="16.4" height="32.8" fill="#6692a6"/>
+	</g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+</svg>
```

Comment:
This looks kinda verbose, maybe you can optimise this image with [svgo](https://github.com/svg/svgo) or a similar tool? (Also see other SVG assets)